<launch>
    <include file="$(find office_test_environment)/launch/office.launch"/>

    <node name="map_server" pkg="map_server" type="map_server" args="$(find robot_assisted_cleaning_solution)/office_map.yaml" output="screen">
		<param name="frame_id" value="map" />
	</node>
    
    <include file="$(find robot_assisted_cleaning_solution)/launch/robot.launch"/>
    <include file="$(find robot_assisted_cleaning_solution)/launch/teb.launch"/> 
    <include file="$(find ira_laser_tools)/launch/laserscan_multi_merger.launch" />

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

    <!-- <node pkg="assisted_cleaning_solution" name="smach_assisted_cleaning_solution" type="smach_assisted_cleaning_solution.py"/> -->
    <node pkg="assisted_cleaning_solution" name="select_coords" type="select_coords.py"/>
    <node pkg="assisted_cleaning_solution" name="find_chairs" type="find_chairs.py"/>
    <node pkg="assisted_cleaning_solution" name="group_chairs" type="group_chairs.py"/>
    <node pkg="assisted_cleaning_solution" name="check_chairs_nearby" type="check_chairs_nearby.py"/>
    <node pkg="assisted_cleaning_solution" name="go_to_chair" type="go_to_chair.py"/>
    <node pkg="assisted_cleaning_solution" name="move_around_chair" type="move_around_chair.py"/>
    <node pkg="assisted_cleaning_solution" name="go_to_path" type="go_to_path.py"/>
    <node pkg="assisted_cleaning_solution" name="positioning_for_plinths" type="positioning_for_plinths.py"/>
    <node pkg="assisted_cleaning_solution" name="follow_plinths" type="follow_plinths.py"/>
    <node pkg="assisted_cleaning_solution" name="make_turn" type="make_turn.py"/>

    <!-- independent nodes -->
    <node pkg="assisted_cleaning_solution" name="lights" type="lights.py"/>
    <!-- <node pkg="assisted_cleaning_solution" name="microcontroller_communication" type="microcontroller_communication.py"/> -->

</launch>